src/pages/journal/components/journal-navigation/journal-navigation.scss

```diff
@@ -0,0 +1,44 @@
+journal-navigation {
+  margin: 24px 12px;
+
+  .navigation-button {
+    margin-top: 12px;
+    font-size: 20px;
+    font-weight: 900;
+    letter-spacing: 0.19px;
+    line-height: 25px;
+  }
+
+  .next-navigation-button {
+    @extend .navigation-button;
+    float: right;
+  }
+
+  .navigation-arrow {
+    width: 48px;
+    height: 48px;
+    font-weight: bolder;
+    border-radius: 50%;
+    color: #000;
+    line-height: 48px;
+    text-align: center;
+    background: #fff;
```

Comment:
can we use the colour variables for the background and font colour. I would also confirm if the text should be pure black or gds black?

Reply:
The colour codes came from Abstract, and also confirmed with Nad. And the pure black and white colours are not in the variables at the moment.

Considering that we are going to have a session with Oli on this Friday and clean up the colour variables. I would not introduce anything to it right now.